docs/elements/hx-search/index.html

```diff
@@ -0,0 +1,75 @@
+---
+title: <hx-search>
+also:
+  components/search: Search
+---
+{% extends 'element.njk' %}
+{% block content %}
+  <section>
+    <p>
+      The custom
+      <code>{{page.title}}</code> element renders search control that accepts user input.
```

Comment:
ohhhh new feature(`page.title`)? are we going to implement throughout helix-ui?

Reply:
I didn't even notice this. It was there when I picked up the branch, but I like it!